Packs/CommonScripts/Scripts/CheckSenderDomainDistance/CheckSenderDomainDistance_test.py

```diff
@@ -0,0 +1,42 @@
+import demistomock as demisto
+import pytest
+
+
+@pytest.mark.parametrize('str1, str2, expected_results', [
+    ('aaa', 'aaa', 0),
+    ('aba', 'baba', 1),
+    ('kitten', 'sitting', 3)
+])
+def test_levenshtein(str1, str2, expected_results):
+    """
+        Given
+        - Two strings.
+        When
+        - Calling levenshtein function.
+        Then
+        - Return the Levenshtein distance (int).
+    """
+    from CheckSenderDomainDistance import levenshtein
+
+    assert levenshtein(str1, str2) == expected_results
+
+
+def test_main(mocker):
```

Comment:
Missing tests for the errors:

- no domain
- no domain in sender
- no sender in email